implementations/rust/ockam/ockam_api/src/nodes/models/policy.rs

```diff
@@ -39,3 +39,80 @@ impl Expression {
         &self.expr
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use quickcheck::{quickcheck, Arbitrary, Gen, TestResult};
+
+    use ockam_abac::{Action, Expr};
+
+    use crate::nodes::models::policy::{Expression, PolicyList};
+    use crate::schema::tests::validate_with_schema;
+
+    quickcheck! {
+        fn policy_list(g: PolicyList) -> TestResult {
+            validate_with_schema("policy_list", g)
+        }
+    }
+
+    quickcheck! {
+        fn policy(g: Expression) -> TestResult {
+            validate_with_schema("expression", g)
+        }
+    }
+
+    impl Arbitrary for PolicyList {
+        fn arbitrary(g: &mut Gen) -> Self {
```

Comment:
It would be good to compose an `Arbitrary for Action` and an `Arbitrary for Expr` to create this `Arbitrary`

Reply:
agree, but the _validate_with_schema_ test util is in ockam_api and Expr and Action are in ockam_abac

I see no easy and elegant solution to this (but maybe my rust knowledge is incomplete here)
I tried to move the util to ockam_abac, but is it possible to have dependencies on test modules?
otherwise I'd have to copy, which I don't like, obviously